cylc/flow/scheduler.py

```diff
@@ -699,8 +699,8 @@ async def start(self):
             self.server.thread.start()
             barrier.wait()
 
-            await self.configure()
             self._configure_contact()
+            await self.configure()
```

Comment:
Unrelated: Swap the order of these two so that the contact file gets written earlier.

This reduces the window of opportunity for two schedulers to start up for the same workflow. Taking a look at the contact file fields, I can't see anything set during `Scheduler.configure` which is used in `_configure_contact` so the order should be arbitrary.